site/styles/_buttons.scss

```diff
@@ -244,3 +244,26 @@
     border-color: $gray-200;
   }
 }
+
+s72-donate-button:not(:empty) {
```

Comment:
Still not convinced that you need that `:not(:empty)` lol

Reply:
i did remove it but the styles didnt quite work, it think it was the background that was getting overriden

Reply:
Yeah that `:not(:empty)` will be giving relish extra specificity.

FYI original intent was to avoid having a random 60px transparent square when disabled in uber. I shouldve added a comment.